TTS/bin/synthesize.py

```diff
@@ -161,6 +161,10 @@ def main():
         default=None,
     )
     parser.add_argument("--gst_style", help="Wav path file for GST stylereference.", default=None)
+    parser.add_argument(
+        "--capacitron_reference_wav", type=str, help="Wav path file for Capacitron prosody reference.", default=None
+    )
+    parser.add_argument("--capacitron_reference_text", type=str, help="Transcription of the reference.", default=None)
```

Comment:
and we can also find a generic name for `capacitron_reference_text` . Maybe `reference_text` if it makes sense. But I need to check the paper again to see what it does actually.

Reply:
Reference text is the transcription of the reference wav you feed when you want to do prosody transfer - it's needed for the reference encoding

Reply:
then maybe call it `reference_wav` and `reference_text`?